postman/threads-api.postman_collection.json

```diff
@@ -1387,18 +1542,21 @@
 										"method": "GET",
 										"header": [],
 										"url": {
-											"raw": "{{api_host}}/me?fields={{fields_profile}}",
+											"raw": "https://graph.threads.net/me?fields=id,username,name,threads_profile_picture_url,threads_biography",
+											"protocol": "https",
 											"host": [
-												"{{api_host}}"
+												"graph",
+												"threads",
+												"net"
 											],
 											"path": [
 												"me"
 											],
 											"query": [
 												{
 													"key": "fields",
-													"value": "{{fields_profile}}",
-													"description": "id: Threads user ID. This is returned by default.\n\nusername: Handle or unique username on Threads. This is the same as the username on Instagram.\n\nthreads_profile_picture_url: URL of the user's profile picture on Threads.\n\nthreads_biography: Biography text on Threads profile."
+													"value": "id,username,name,threads_profile_picture_url,threads_biography",
```

Comment:
Let's keep this as `{{fields_profile}}` if possible so that it's easy to keep up-to-date.